Rationale.md

```diff
@@ -106,10 +106,16 @@ tradeoffs.
 
 ## Resizing
 
-Implementations provide a `page_size` operator which allows them to efficiently
-map the underlying OS's capabilities to the WebAssembly application, as well as
-to communicate their own implementation details in a useful manner to the
-developer.
+To allow efficient virtual-memory based techniques for bounds checking, memory
+sizes are required to be page-aligned.
+For portability across a range of CPU architectures and operating systems,
+WebAssembly defines a fixed page size.
+Programs can depend on this fixed page size and still remain portable across all
+WebAssembly engines which employ efficient virtual memory techniques for bounds\
+checking.
```

Comment:
This sentence reads a little awkward. Possible suggestion: Programs can depend on this fixed page size and still remain portable across all WebAssembly engines [Full Stop]. And, engines can rely on this invariant to employ efficient virtual memory techniques for bounds checking.